tach/cache/access.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+
+import os
+import uuid
+
+from tach.cache.setup import resolve_dot_tach, resolve_project_dir
+
+
+def get_uid() -> uuid.UUID:
+    project_root = resolve_project_dir()
+    if not os.path.exists(project_root / ".tach" / "tach.info"):
```

Comment:
I think this works: `if not (project_root / ".tach" / "tach.info").exists():`

Reply:
nice, updated/tested